GliaWidgets/UIColor+Extensions.swift

```diff
@@ -10,6 +10,19 @@ extension UIColor {
         )
     }
 
+    var hex: String {
+        var r: CGFloat = 0
+        var g: CGFloat = 0
+        var b: CGFloat = 0
+        var a: CGFloat = 0
+
+        getRed(&r, green: &g, blue: &b, alpha: &a)
+
+        let rgb: Int = (Int)(r * 255) << 16 | (Int)(g * 255) << 8 | (Int)(b * 255) << 0
```

Comment:
I think in this case it is better to use `UInt` for representation of color hex since `UInt.max` provides twice wider range than `Int.max` on positive side,  and there's no need for negative numbers.